great_expectations/agent/actions/agent_action.py

```diff
@@ -0,0 +1,30 @@
+from abc import abstractmethod
+from typing import Generic, Sequence, TypeVar
+
+from pydantic import BaseModel
+
+from great_expectations.agent.models import Event
+from great_expectations.data_context import CloudDataContext
+
+
+class CreatedResource(BaseModel):
+    type: str
+    id: str
+
+
+class ActionResult(BaseModel):
+    id: str
+    type: str
+    created_resources: Sequence[CreatedResource]
+
+
+_TEvent = TypeVar("_TEvent", bound=Event)
```

Comment:
The convention we've been using for type vars is to have a `T` suffix.
```suggestion
_EventT = TypeVar("_EventT", bound=Event)
```

Reply:
addressed in [5a5b74f](https://github.com/great-expectations/great_expectations/pull/7882/commits/5a5b74f70ece32713e37ee247935342ed86606cc)